packages/nyc-test-coverage/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@temporalio/nyc-test-coverage",
+  "version": "1.0.1",
+  "description": "Temporal.io SDK code coverage integration",
+  "main": "lib/index.js",
+  "types": "./lib/index.d.ts",
+  "keywords": [
+    "temporal",
+    "workflow",
+    "testing",
+    "coverage"
+  ],
+  "author": "Temporal Technologies Inc. <sdk@temporal.io>",
+  "license": "MIT",
+  "dependencies": {
+    "@temporalio/workflow": "file:../workflow",
+    "@temporalio/worker": "file:../worker",
+    "istanbul-lib-coverage": "3.2.0"
+  },
+  "devDependencies": {
+    "@types/istanbul-lib-coverage": "2.0.4"
```

Comment:
Just noticed this, why use exact versions instead of caret?

Reply:
Best practice. I'll switch to `^` for consistency with other packages.